homeassistant/components/yale_smart_alarm/alarm_control_panel.py

```diff
@@ -80,7 +81,7 @@ async def async_set_alarm(self, command: str, code: str | None = None) -> None:
                 )
         except YALE_ALL_ERRORS as error:
             raise HomeAssistantError(
-                f"Could not set alarm for {self._attr_name}: {error}"
+                f"Could not set alarm for {self.coordinator.entry.data[CONF_NAME]}: {error}"
```

Comment:
Side note: Consider setting the username as coordinator name and use instead. The config entry data name item doesn't seem unique per config entry.